src/main/kotlin/no/nav/syfo/tilgang/TilgangApi.kt

```diff
@@ -4,23 +4,56 @@ import io.ktor.http.*
 import io.ktor.server.application.*
 import io.ktor.server.response.*
 import io.ktor.server.routing.*
+import no.nav.syfo.application.api.auth.isMissingNAVIdent
 import no.nav.syfo.util.getBearerHeader
 import no.nav.syfo.util.getCallId
 
 const val tilgangApiBasePath = "/api/tilgang"
+const val enhetNr = "enhetNr"
 
 fun Route.registerTilgangApi(
     tilgangService: TilgangService,
 ) {
     route(tilgangApiBasePath) {
         get("/navident/syfo") {
+            val callId = call.getCallId()
             val token = call.getBearerHeader()
                 ?: throw IllegalArgumentException("Failed to check syfo tilgang for veileder. No Authorization header supplied")
+            if (token.isMissingNAVIdent()) {
+                throw IllegalArgumentException("Failed to check enhetstilgang for veileder. No NAV ident in token")
+            }
+
+            val tilgang = tilgangService.hasTilgangTilSyfo(
+                token = token,
+                callId = callId,
+            )
+
+            if (tilgang.harTilgang) {
+                call.respond(tilgang)
+            } else {
+                call.respond(
+                    status = HttpStatusCode.Forbidden,
+                    message = tilgang
+                )
+            }
+        }
+
+        get("/navident/enhet/{$enhetNr}") {
```

Comment:
Her hadde @juneih lagt på en `// TODO: sjekk claims`, men jeg vet ikke hva som menes med det 🤔 
Vi sjekke `aud`-claim i `installJwtAuthentication`. Altså at den som kaller istilgangskontroll faktisk har istilgangskontroll i sin audience claim.

Reply:
Ja, det er authenticate(JwtIssuerType.INTERNAL_AZUREAD.name) i Application.apiModule som sørger for dette - bør være ok.